modules/activiti-engine/src/test/resources/org/activiti/engine/test/api/event/ActivityEventsTest.testActivityMessageBoundaryEventsOnSubProcess.bpmn20.xml

```diff
@@ -29,7 +29,7 @@
     </sequenceFlow>
     <sequenceFlow id="cloudsequenceflow3" isImmediate="false" sourceRef="cloudformtask2" targetRef="cloudendnoneevent1">
     </sequenceFlow>
-    <sequenceFlow id="cloudsequenceflow4" isImmediate="false" sourceRef="subProcess" targetRef="cloudendnoneevent2">
+    <sequenceFlow id="cloudsequenceflow4" isImmediate="false" sourceRef="subProcess" targetRef="cloudformtask2">
```

Comment:
This change is invalid (cloudendnoneevent2 will have no source sequence) and doesn't seem needed. Can you revert this change?